HHS/Head-Start-TTADP
# Hide the course select if there are no courses

## Description of change
Just because of sequencing/pipeline, it's likely we will need to push the course select to main before we can reliably put courses in the database.

## How to test
Create a report in a local environment where you have no courses in the course table and confirm that when creating an objective the api returns an empty array, the course select doesn't render, and you can create one unimpeded.

## Issue(s)

## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
